petclinic-frontend/src/features/customers/components/CustomerDetails.css

```diff
@@ -1,42 +1,80 @@
 .customer-details-card {
   background-color: #ffffff;
-  border-radius: 8px;
-  box-shadow: 0 4px 20px rgba(0, 0, 0, 0.1);
-  padding: 30px;
+  border-radius: 12px;
+  box-shadow: 0 8px 30px rgba(0, 0, 0, 0.1);
+  padding: 40px;
   margin: 40px auto;
-  max-width: 600px;
+  max-width: 900px;
   text-align: left;
 }
 
 .customer-details-card h2 {
   margin-bottom: 20px;
   color: #333;
+  font-size: 2em;
+  text-align: center;
+}
+
+.customer-details-container {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+.section {
+  flex: 1 1 30%;
+  margin: 15px;
+  padding: 20px;
+  border-radius: 8px;
+  background-color: #f8f9fa;
+  box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1);
+  transition: transform 0.3s, box-shadow 0.3s;
+}
+
+.section:hover {
+  transform: translateY(-3px);
+  box-shadow: 0 4px 20px rgba(0, 0, 0, 0.15);
+}
+
+.section h3 {
+  color: #007bff;
+  margin-bottom: 15px;
   font-size: 1.5em;
 }
 
-.customer-details-card p {
-  margin: 12px 0;
+.section p, .section ul {
```

Comment:
NITPICK: Although really not important, I think you should still be using classes such as customer-details-section to make sure only the css you want to affect is being affected